docs/serving/installing-logging-metrics-traces.md

```diff
@@ -228,6 +228,8 @@ Knative.
 
 ## End to end traces
```

Comment:
```suggestion
## End to end request tracing

You can choose from one of the following options to enable request tracing in your Knative Serving cluster.

**Important**: Your cluster supports only a single request trace tool. If you want to replace a currently installed request trace tool, you must first uninstall that tool before installing the new tool.
```

Its not clear from #3696 but Im assuming since the port numbers are the same, that multiple tools is not supported.